package proto_ff;

import "yd_fieldoptions.proto";


message handbookhandbookattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int64 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message handbookhandbook
{
	optional int64 handbookID = 1[(yd_fieldoptions.field_cname) = "图鉴id"];
	optional int64 itemId = 2[(yd_fieldoptions.field_cname) = "关联物品id"];
	optional string handbookName = 3[(yd_fieldoptions.field_cname) = "图鉴名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 handbookType = 4[(yd_fieldoptions.field_cname) = "图鉴类型"];
	optional int32 handbookQuality = 5[(yd_fieldoptions.field_cname) = "图鉴品质"];
	optional int32 handbookLv = 6[(yd_fieldoptions.field_cname) = "图鉴等级上限"];
	optional int32 handZ = 7[(yd_fieldoptions.field_cname) = "图鉴组"];
	optional int64 starId = 8[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string starNum = 9[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 300];
	repeated handbookhandbookattributeDesc attribute = 10[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 4];
}


message Sheet_handbookhandbook
{
	repeated handbookhandbook handbookhandbook_List = 1[(yd_fieldoptions.field_arysize)=500];
}

message handbookfateattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int64 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message handbookfate
{
	optional int64 fateID = 1[(yd_fieldoptions.field_cname) = "羁绊id"];
	optional string fateName = 2[(yd_fieldoptions.field_cname) = "羁绊名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 starLv = 3[(yd_fieldoptions.field_cname) = "升星等级上限"];
	repeated int64 fateHand = 4[(yd_fieldoptions.field_cname) = "羁绊图鉴", (yd_fieldoptions.field_arysize) = 4];
	repeated handbookfateattributeDesc attribute = 5[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 4];
}


message Sheet_handbookfate
{
	repeated handbookfate handbookfate_List = 1[(yd_fieldoptions.field_arysize)=40];
}

message handbookattr
{
	optional int64 attID = 1[(yd_fieldoptions.field_cname) = "属性组id"];
	repeated int32 att = 2[(yd_fieldoptions.field_cname) = "属性等级", (yd_fieldoptions.field_arysize) = 76];
}


message Sheet_handbookattr
{
	repeated handbookattr handbookattr_List = 1[(yd_fieldoptions.field_arysize)=500];
}
